test/EFCore.PG.FunctionalTests/StoreGeneratedFixupNpgsqlTest.cs

```diff
@@ -40,6 +40,14 @@ protected override void MarkIdsTemporary(DbContext context, object dependent, ob
             entry.Property("Id1").IsTemporary = true;
             entry.Property("Id2").IsTemporary = true;
 
+            foreach (var property in entry.Properties)
+            {
+                if (property.Metadata.IsForeignKey())
+                {
+                    property.IsTemporary = true;
+                }
+            }
```

Comment:
nit: These brackets could be removed.

Reply:
Yeah, that's just a copy-paste from SQL Server code. For stuff like this which I really don't touch, I prefer to leave it exactly like the source to make diffing easier.